src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAHelper.h

```diff
@@ -92,33 +94,58 @@ namespace OpenMS
     */
     OPENMS_DLLAPI void adjustExtractionWindow(double& right, double& left, const double& mz_extract_window, const bool& mz_extraction_ppm);
 
-    /// compute the b and y series masses for a given AASequence
-    OPENMS_DLLAPI void getBYSeries(const AASequence& a,
-                                   std::vector<double>& bseries,
-                                   std::vector<double>& yseries,
+    /// compute the prefix and suffix series masses for a given AASequence
+    OPENMS_DLLAPI void getPrefixSuffixSeries(const AASequence& a,
+                                   std::vector<double>& prefixSeries,
+                                   std::vector<double>& suffixSeries,
```

Comment:
This new name is kind of confusing, however I am not sure what is better. Possibly at least include that for b and y series in the comment.

Reply:
I would prefer not renaming this function if possible